postgresql/ref/grant.xml

```diff
@@ -263,38 +263,37 @@ GRANT <replaceable class="parameter">nom_role</replaceable> [, ...] TO <replacea
     <literal>TRUE</literal>, donc <literal>WITH ADMIN OPTION</literal>
     est un synonyme pour <literal>WITH ADMIN TRUE</literal>. Lors de la
     modification d'une appartenance existante, l'omission d'une option
-    résultat en la conservation de la valeur actuelle.
+    résulte en la conservation de la valeur actuelle.
    </para>
 
    <para>
-    L'option <literal>ADMIN</literal> autorise le membre
-    en octroyer l'appartenance à d'autres rôles, et la révoquer. Sans cette
-    option, les utilisateurs ordinaires ne peuvent pas le faire. Un rôle ne
-    dispose pas de l'option <literal>WITH ADMIN OPTION</literal> lui-même Les
+    L'option <literal>ADMIN</literal> autorise le membre à octroyer
+    l'appartenance à d'autres rôles, et la révoquer. Sans cette option, les
+    utilisateurs ordinaires ne peuvent pas le faire. Un rôle ne dispose pas de
+    l'option <literal>WITH ADMIN OPTION</literal> sur lui-même. Les
     superutilisateurs de la base peuvent donner ou révoquer l'appartenance à un
-    rôle pour tout personne. Cette option est à
-    <literal>FALSE</literal> par défaut.
+    rôle pour toute personne. Cette option est à <literal>FALSE</literal> par
+    défaut.
    </para>
 
    <para>
-    L'option <literal>INHERIT</literal> contrôle le statut d'héritage
-    du nouveau membre&nbsp;; voir <xref linkend="role-membership"/>
-    pour les détails sur l'héritage. Si elle est configurée à
-    <literal>TRUE</literal>, elle fait que le nouveau membre hérite
-    du rôle donné. Si elle est configurée à
-    <literal>FALSE</literal>, le nouveau membre n'hérite pas. Sans
-    indication lors de la création d'un nouveau rôle, le défaut est
-    l'ajout de l'attribut d'héritage au rôle ajouté.
+    L'option <literal>INHERIT</literal> contrôle le statut d'héritage du nouveau
+    membre&nbsp;; voir <xref linkend="role-membership"/> pour les détails sur
+    l'héritage. Si elle est configurée à <literal>TRUE</literal>, elle fait que
+    le nouveau membre hérite du rôle donné. Si elle est configurée à
+    <literal>FALSE</literal>, le nouveau membre n'hérite pas. Sans indication
+    lors de la création d'une nouvelle appartenance de rôle, le défaut est la
+    valeur de l'attribut d'héritage au rôle ajouté.
    </para>
 
    <para>
     L'option <literal>SET</literal>, si elle est configurée à
-    <literal>TRUE</literal>, permet au membre de modifier le rôle donné
-    en utilisant l'instruction <link linkend="sql-set-role"><command>SET
-    ROLE</command></link>. Si un rôle est un membre indirect d'un autre
-    rôle, il peut utiliser <literal>SET ROLE</literal> seulement s'il existe
-    une chaîne de dons pour lesquels chacun est configuré à <literal>SET
-    TRUE</literal>. Cette option vaut <literal>TRUE</literal> par défaut.
+    <literal>TRUE</literal>, permet au membre de changer vers le rôle donné en
```

Comment:
Originale: dans REL_16_STABLE:

``` xml
   The <literal>SET</literal> option, if it is set to
   <literal>TRUE</literal>, allows the member to change to the granted
   role using the
   <link linkend="sql-set-role"><command>SET ROLE</command></link>
   command. If a role is an indirect member of another role, it can use
   <literal>SET ROLE</literal> to change to that role only if there is a
   chain of grants each of which has <literal>SET TRUE</literal>.
   This option defaults to <literal>TRUE</literal>.
```